src/components/custom-aggrid/custom-aggrid-header.js

```diff
@@ -128,18 +118,51 @@ const CustomHeaderComponent = ({
         debouncedUpdateFilter([{ text: selectedFilterData, type: newType }]);
     };
 
-    const handleSortChange = useCallback(() => {
-        let newSort = null;
-        if (!isColumnSorted || !sortWay) {
-            newSort = 1;
-        } else if (sortWay > 0) {
-            newSort = -1;
-        }
+    const handleSortChange = useCallback(
```

Comment:
in this function : `const handleSortChange = useCallback(` 
Can you add comments to help understand the logic ? Maybe something like "it was sorted ASC, we click so it goes DESC", etc.